specs-go/v1/manifest.go

```diff
@@ -37,6 +37,10 @@ type Manifest struct {
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
 
-// ScratchDigestSHA256 is the digest of a blob with content of `{}` (size of 2)
-const ScratchDigestSHA256 = `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a`
-const ScratchDigestData = `{}`
+const (
+	// ScratchDigestSHA256 is the digest of a blob with content of `{}` (size of 2).
+	ScratchDigestSHA256 = `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a`
```

Comment:
IMO, if we keep this field, we should drop `SHA256` from the end of it too, but as discussed in the call today, now that we have an explicit media type for scratch from #1043, we can probably revisit https://github.com/opencontainers/image-spec/pull/1023#issuecomment-1428455309 and have a proper `Descriptor` object again (and not really need a function for it).